js/services/vCardProperties.js

```diff
@@ -66,6 +66,14 @@ angular.module('contactsApp')
 			readableName: t('contacts', 'Birthday'),
 			template: 'date'
 		},
+		anniversary: {
+			readableName: t('contacts', 'Anniversary'),
+			template: 'date'
+		},
+		deathdate: {
+			readableName: t('contacts', 'Date of Death'),
```

Comment:
Correct capitalization should be »Date of death«, only first character capitalized.